code/game/objects/structures/inflatable.dm

```diff
@@ -444,7 +444,7 @@
 			A.ex_act(severity++)
 	..()
 
-/obj/structure/inflatable/shelter/container_resist(mob/user)
+/obj/structure/inflatable/shelter/container_resist(var/mob/user,var/turf/dest)
```

Comment:
We generally prefer not using `var/` here

Reply:
(respectfully disagree)

Reply:
Is it finally time to write a style guide??

Reply:
do it

Reply:
Been doing that for 5+ years, why the change?

Reply:
It's just unnecessary, just like `src.`